dispatch.go

```diff
@@ -40,13 +40,20 @@ func (d *dispatch) Call(ctx context.Context, name string, arg interface{}, res i
 	profiler := d.log.StartProfiler("call %s", name)
 	defer profiler.Stop()
 
+	select {
```

Comment:
They're independent, but they're both closed by the transport. It'll stop correctly as is.